Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
@chrmoritz here's grafana node 8 + npm 4.2.0 https://gist.github.com/ilovezfs/c1f95d7e5d50dd4ed0cfc9acbfff2c18

As you can see, no problems.